src/rpc/ckb.rs

```diff
@@ -81,3 +83,152 @@ crate::jsonrpc!(pub struct CkbRpcClient {
     pub fn update_main_logger(&mut self, config: MainLoggerConfig) -> ();
     pub fn set_extra_logger(&mut self, name: String, config_opt: Option<ExtraLoggerConfig>) -> ();
 });
+
+fn transform_cycles(cycles: Option<Vec<ckb_jsonrpc_types::Cycle>>) -> Vec<Cycle> {
+    if let Some(cycles) = cycles {
+        cycles.into_iter().map(|c| c.into()).collect()
+    } else {
+        vec![]
+    }
+}
+
+impl CkbRpcClient {
+    pub fn get_packed_block(&mut self, hash: H256) -> Result<Option<JsonBytes>, crate::RpcError> {
+        self.post("get_block", (hash, Some(Uint32::from(0u32))))
+    }
+
+    // turn block response into BlcokView and cycle vec
+    fn blockresponse2view(
```

Comment:
suggest to rename the fn to `transform_block_view_with_cycle`, follow naming convention like `transform_cycles`, and move this fn to a outside helper fn instead of a `CkbRpcClient` inner fn.

also suggest to rename the fn `blockresponse2bytes` to `transform_json_bytes_with_cycle`